### PR TITLE
Automatically approve Dependabot PRs

### PR DESCRIPTION
**Pull request by jtbandes**
**User-Facing Changes**
None

**Description**
Automatically approve Dependabot PRs when they are opened.

A small step towards FG-3299

### CODE REVIEW

File: .github/workflows/dependabot-approve.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Dependabot
```
Comment (defunctzombie) [inline .github/workflows/dependabot-approve.yml:1]: https://github.com/foxglove/studio/blob/main/.github/workflows/dependabot-fix.yml#L10 Is it ok to use the same name for two workflows?

Please add a comment describing the goals of this workflow.

Comment (jtbandes) [reply .github/workflows/dependabot-approve.yml:1]: https://stackoverflow.com/questions/70701626/github-actions-same-workflow-name-different-branches-which-takes-priority#:~:text=Two%20workflows%20with%20the%20same,appear%20using%20the%20same%20name.

**Review by defunctzombie — APPROVED** [review]
Not sure what I think about stack-overflow being the "citation" for this behavior but we can see how it goes. Also would be fine to call the workflows different names if we run into problems.

**Comment by jtbandes** [comment]
📿

**Pull request merged**